scout/adapter/mongo/variant_loader.py

```diff
@@ -338,6 +337,7 @@ def load_variant_bulk(self, variants):
         except (DuplicateKeyError, BulkWriteError) as err:
             # If the bulk write is wrong there are probably some variants already existing
             # In the database. So insert each variant
+            LOG.debug("Bulk insertion failed - attempting separate upsert")
```

Comment:
Why debug and not warning here?

Reply:
Good point - I actually ran the "test"with debug. Will fix!